### PR TITLE
WFLY-19518 Nightly 'Standard - MP Certifications - JDK 11' CI job fails

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-19518

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1132,7 +1132,7 @@
                             <goals>
                                 <goal>enforce</goal>
                             </goals>
-                            <phase>compile</phase>
+                            <phase>none</phase>
```
Comment (bstansberry) [inline pom.xml:1135]: This turns off enforcement for the entire build. You need to add this plugin config to testsuite/integration/microprofile-tck/certification/pom.xml and set the phase to 'none' there.

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (RanabirChakraborty) [reply pom.xml]: Thanks! for the review, fixed it.

**Comment by bstansberry** [comment]
@RanabirChakraborty Thank your for https://ci.wildfly.org/viewLog.html?buildId=445642&tab=buildResultsDiv&buildTypeId=WF_MainNightlyJobs_StandardMpCertificationsJdk11

**Comment by bstansberry** [comment]
Thanks @RanabirChakraborty

**Pull request merged**